src/js/breadcrumbs/specs/breadcrumbs-spec.js

```diff
@@ -0,0 +1,44 @@
+(function (define) {
+    'use strict';
+
+    define(['jquery',
+            'underscore',
+            '../breadcrumbs-view.js',
+            '../breadcrumbs-model.js'
+           ],
+           function($, _, BreadcrumbsView, BreadcrumbsModel) {
+
+               describe('Breadcrumbs component', function () {
+                   var model, view;
+
+                   var testBreadcrumbs = function (breadcrumbs) {
+                       model.set('breadcrumbs', breadcrumbs);
+                       expect(view.$('nav.breadcrumbs').length).toBe(1);
+                       _.each(view.$('.nav-item'), function (el, index) {
+                           expect($(el).attr('href')).toEqual(breadcrumbs[index].url);
+                           expect($(el).text()).toEqual(breadcrumbs[index].title);
+                       });
+                   };
+
+                   beforeEach(function () {
+                       model = new BreadcrumbsModel();
+                       view = new BreadcrumbsView({
+                           model: model
+                       });
+                   });
+
+                   it('does not show breadcrumbs by default', function () {
+                       expect(view.$el.html()).not.toContain('<nav class="breadcrumbs">');
+                   });
+
+                   it('shows breadcrumbs if they are supplied', function () {
```

Comment:
It would be good to test a variety of different breadcrumbs. Look at using the ``SpecHelpers.withData`` function so that you can do this in a declarative way, e.g. https://github.com/edx/edx-ui-toolkit/blob/master/src/js/utils/specs/html-utils-spec.js#L101

Some tests I would include:
 - rendering a single breadcrumb
 - rendering two items in a breadcrumb
 - rendering Unicode characters
 - rendering labels that include HTML to verify that they are safely escaped